controllers/users.js

```diff
@@ -558,6 +559,15 @@ const nonVerifiedDiscordUsers = async (req, res) => {
   return res.json(data);
 };
 
+const setInDiscordScript = async (req, res) => {
+  try {
+    await setInDiscordFalseScript();
```

Comment:
- `setInDiscordFalseScript` is this a script?

Reply:
it is a function written in a services file which works as a script, it gets all the users in our db and sets a role of in_discord false to them.